external/avro/src/test/scala/org/apache/spark/sql/avro/AvroCodecSuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.avro
+
+import org.apache.spark.sql.execution.datasources.FileSourceCodecSuite
+import org.apache.spark.sql.internal.SQLConf
+
+class AvroCodecSuite extends FileSourceCodecSuite {
+
+  override def format: String = "avro"
+  override val codecConfigName: String = SQLConf.AVRO_COMPRESSION_CODEC.key
+  override protected def availableCodecs = Seq("uncompressed", "deflate", "snappy",
+    "bzip2", "xz", "zstandard")
```

Comment:
do we support `zstandard` for Avro? don't see it here: https://spark.apache.org/docs/latest/sql-data-sources-avro.html

Reply:
oh I see it does, just the doc needs update.

Reply:
yea, seems we don't update the doc yet.

Reply:
We use `zstd` in parquet. Is it possible to use the same name?

Reply:
I think it is because in avro, it is named exactly `zstandard`.

https://github.com/apache/avro/blob/master/lang/java/avro/src/main/java/org/apache/avro/file/DataFileConstants.java#L41